docs/README.md

```diff
@@ -313,6 +313,45 @@ Notes:
 * All regular expressions are case-sensitive by default. If you don't want to differentiate between upper and lower-case letters, use non-capturing groups with a flag: `(?flags:exp)`, for example: `(?i:my_case_insensitive_regex)`
 * LogDNA also provides post-ingestion <a href="https://docs.logdna.com/docs/excluding-log-lines" target="_blank">exclusion rules</a> to control what log data is displayed and stored in LogDNA.
 
+### Configuration for Kubernetes Metadata Filtering
```

Comment:
Move this whole section to docs/KUBERNETES.md

Reply:
Done.